### DISCUSSION TITLE
동영상 촬영을 예제 추가해 주실 수 있나요?

### DISCUSSION THREAD

**Discussion by earthboom** [naverz/zepeto-studio-kor#894] [5. Q&A]
깃 허브에 있는 스크린샷 예제 처럼, 동영상 촬영도 예제로 추가해 주실 수 있나요?

가이드를 참고하고 있지만, 쉽지 않아서 문의 드려요.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 동영상 촬영 예제 준비 중에 있습니다.
조금만 기다려 주세요~
감사합니다!
